packages/@react-spectrum/illustratedmessage/docs/IllustratedMessage.mdx

```diff
@@ -0,0 +1,97 @@
+<!-- Copyright 2020 Adobe. All rights reserved.
+This file is licensed to you under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License. You may obtain a copy
+of the License at http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software distributed under
+the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+OF ANY KIND, either express or implied. See the License for the specific language
+governing permissions and limitations under the License. -->
+
+import {Layout} from '@react-spectrum/docs';
+export default Layout;
+
+import docs from 'docs:@react-spectrum/illustratedmessage';
+import {HeaderInfo, PropTable} from '@react-spectrum/docs';
+import packageData from '../package.json';
+
+```jsx import
+import {Content} from '@react-spectrum/view';
+import {Heading} from '@react-spectrum/typography';
+import {IllustratedMessage} from '@react-spectrum/illustratedmessage';
+```
+
+# IllustratedMessage
+
+<p>{docs.exports.IllustratedMessage.description}</p>
+
+<HeaderInfo
+  packageData={packageData}
+  componentNames={['IllustratedMessage']} />
```

Comment:
I couldn't find the spectrum docs on this. I did notice that spectrum-css had a broken link to spectrum docs about this. 

https://opensource.adobe.com/spectrum-css/components/illustratedmessage/
broken => https://spectrum.adobe.com/page/informational-illustrations/

Reply:
Note - spectrum said they combined all their illustrations to https://spectrum.corp.adobe.com/page/illustration/ I didn't see anything IllustratedMessage on this.